Scripts/Multiplayer/server.gd

```diff
@@ -171,9 +175,14 @@ func broadcast_full_game_setup(game_setup : GameSetupInfo):
 		"name": "fill_game_setup",
 		"setup" : game_setup.to_dictionary(server_username)
 	}
+	# print("sending \n", packet)
 	broadcast(packet)
 
 
+func broadcast_start_game():
+	broadcast(StartGameCommand.create_packet())
```

Comment:
maybe add here an option to pass a config which has not to be imported from input manager -- it will be easier to manage when we will be crating dedicated server